qemu/tests/cfg/balloon_in_use.cfg

```diff
@@ -1,43 +1,51 @@
 - balloon_in_use:
     type = driver_in_use
-    only Windows
     balloon = balloon0
     balloon_dev_devid = balloon0
     balloon_dev_add_bus = yes
     repeat_times = 5
-    ratio = 0.5
     wait_bg_time = 720
-    time_for_video = 600
     start_vm = yes
     kill_vm_on_error = yes
-    check_guest_bsod = yes
     run_bgstress = balloon_stress
-    stress_test = win_video_play
     bg_stress_run_flag = balloon_test
```

Comment:
What's this param for?  Seems meaningless?

Reply:
This param is used to check background test status

Reply:
I checked the code in driver_in_use.py, seems it will always be reassigned with "False" in line 56. I doubt if it is really useful.

Reply:
It will set to 1 in the background test:
balloon_check.py.
    self.env["balloon_test"] = 1

Reply:
The key should be "bg_stress_run_flag" ?

Reply:
Both should be ok.
I don't know whether self.env["balloon_test"] come first or bg_stress_run_flag come first.
But I guess original author just want to connect the flag with the balloon class object:
    balloon_test = BallooningTestWin(test, params, env)
    balloon_test = BallooningTestLinux(test, params, env)

Reply:
And if another .cfg did not set bg_stress_run_flag, balloon_check will fail.
So it's better to use self.env["balloon_test"] = 1.
Of course we set a default value to it if no pram is assigned, but it's meaningless due to:
1). if bg stress is not related to balloon_check, self.env[bg_stress_run_flag] = 1 is not correct;
2). if bs stress is related to balloon_check, then bg_stress_run_flag must be balloon_test, no need to considering other values.